brigade.js

```diff
@@ -137,17 +143,36 @@ function testCLI(e, p) {
 
   // Setup kubeconfig, docker login, run tests
   goTest.tasks.push(
-      "mkdir -p ${HOME}/.kube",
-      'echo "${kubeconfig}" > ${HOME}/.kube/config',
-      `docker login ${p.secrets.dockerhubRegistry} \
-      -u ${p.secrets.dockerhubUsername} \
-      -p ${p.secrets.dockerhubPassword}`,
-      `REGISTRY=${p.secrets.dockerhubOrg} make test-cli`
+    "mkdir -p ${HOME}/.kube",
+    'echo "${kubeconfig}" > ${HOME}/.kube/config',
+    `docker login ${p.secrets.dockerhubRegistry} \
+    -u ${p.secrets.dockerhubUsername} \
+    -p ${p.secrets.dockerhubPassword}`,
+    `REGISTRY=${p.secrets.dockerhubOrg} make test-cli`
   );
 
   return goTest;
 }
 
+function publishExamples(e, p) {
+  var examplePublisher = new GoJob(`${projectName}-publish-examples`);
+  // Enable docker so that the daemon can be used for duffle commands invoked by test-cli
```

Comment:
nit: there is no duffle only zuul

Reply:
Oh!  Good catch.  I thought I rm'd mention of `duffle` in these comments 😓 .   Lemme fix in a follow-up PR.